architecture-docs/adr-003.md

```diff
@@ -114,56 +114,60 @@ of the project.
   }
   ```
   
-- A complete commit would be like this:
+- The complete block commit processing will be done as follows at the end:
   
   ```rust
   flush_staking(&mut kv_store!(self), block_height, staking_buffer);
   flush_kv(&mut self.kvdb, kv_buffer);  // Write to the disk atomically.
   ```
 
-### Chain node state changes
+### Chain Node State Changes
 
-- Remove `AccountStorage` from `ChainNodeApp`.
+- `AccountStorage` will be removed from `ChainNodeApp` (as the node storage will be in the dedicated `COL_TRIE_NODE` column).
 
-- In `ChainState`, change `account_root` to `last_account_block_height`, which records the last block height there's any
-  staking state modified.
+- In `ChainState`, `account_root` will be changed to `last_account_block_height` which records the last block height where any staking state was modified.
```

Comment:
```suggestion
```
Since the hash root needs to be kept (https://github.com/crypto-com/chain/pull/1348/files#r402347319), no need to change this anymore.